### PR TITLE
Card holder search

### PR DESCRIPTION
**Pull request by pkarman**
trello: https://trello.com/c/LNnQtzg9/236-search-not-working-for-card-holder-name-field-queries

note requires re-indexing in order to QA

### CODE REVIEW

File: app/models/proposal.rb [hunk 1]
```diff
@@ -115,6 +112,7 @@ def as_indexed_json(params = {})
       if client_data
         json[:client_data] = client_data.as_indexed_json
       end
+      json[:requester] = requester.display_name
```
Comment (jessieay) [inline app/models/proposal.rb]: while we are working on this, would it be possible to change the "Card holder name" label to read "Card hold name or email address" ? Raphy and Amber asked me what "Card holder name" field was searching last week and I had to look at the code to figure out what it was

**Comment by jessieay** [comment]
code / test looks good! one question about search field label

**Comment by pkarman** [comment]
@jessieay the search label was chosen by @amberwreed -- I have no opinion on it.

cc/ @amberwreed @ericronne @phirefly

**Comment by ericronne** [comment]
I don't yet know enough to offer an educated opinion on the matter :(

File: app/models/proposal.rb (hunk 1 above)
Comment (amberwreed) [inline app/models/proposal.rb]: @pkarman please change the field name to "Requester name or email"
Thanks!

Comment (pkarman) [inline app/models/proposal.rb]: :+1:   18dd97d

**Comment by jessieay** [comment]
@pkarman specs passing first try! you are a magician :crystal_ball:

**Pull request merged**